data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+				accept
+	
+	on complete
+		log "Have learned the basics of the Remnant sign language, and been formally granted access to their worlds after having been genetically tested on each of them."
+		conversation
+			`The last leg of the journey was spent watching some ancient show about a merchant captain making his way through the galaxy with an unlikely crew, re-enacted by a cast of Remnant who sing and sign their way through the dialogs. It seems really familiar, but you can't quite place it. You note that it seems to have a lot of subtext about the dangers of manipulation and government secrecy, especially surrounding genetics.`
+			`	You arrive at the Director's desk and offer to return the datapad to the Director, signing your thanks. She seems pleased at how much you have learned, and replies that you are welcome to keep it for as long as you need it.`
+			`	She pauses for a moment, then signs "You are probably wondering about all the blood tests. Long ago it was decided that being infiltrated by Alpha agents was a big risk, and we couldn't depend on our interstellar network to remain secure. So we implemented a policy where the first time a Remnant is granted formal access to a new planet their purity and identity is verified. Thus even if someone hacked our networks to give themselves a clean identity, they would still be caught when they started working on a new planet. We also don't broadcast that requirement on our networks, simply enforce it locally. It isn't foolproof, but it adds an additional layer of safety."`
+			`	After you finish your conversation with her, you stroll off to look around the spaceport, enjoying the fact that you can now understand a bit of the conversations going on around you. It still takes quite a bit of effort, but it is a good start.`
+
+
+mission "Remnant: Salvage 1"
+	name "Remnant History Updates"
+	description `Find a copy of human history for the Remnant.`
+	source "Viminal"
+	stopover "Alexandria"
+	to offer
+		and
+			has "license: Remnant"
+			has "Remnant: Learning Sign 2: done"
+			random < 75
+			or
+				has "Remnant: Catalytic Ramscoop: done"
+				has "Remnant: Plasma Cannons: done"
+				has "Remnant: Heavy Laser: done"
+	on offer
+		conversation
+			`As you step out into the icy chill of Viminal you get a comlink message from Tali asking if you could meet her in the cafeteria. Replying in the affirmative, you make your way through the crowds and find an empty table. Moments later Tali appears, grabs two trays from the dispenser, and settles down with an air of exhaustion on the chair opposite you.`
+			`	"It is good to see you again, <first>." she sings softly. When you sign a reply she sits up and smiles. "My, you are a quick learner." she signs "Great, this is much faster." `
```

Comment:
Note; This is where I will stop nit-picking on the text for now(Which, I'm guessing you hate, since I'm just mercilessly picking out errors/ putting in my own suggestions)

Reply:
Was there a change in this section?